### PR TITLE
Slight tweaks so we can start adding more support-related stuff here

### PR DESCRIPTION
**Pull request by virginiaulrich**
- Changed support name on main page to Customer Support
- Changed "onboarding" on CE page to "Customer Engineering Onboarding" so it's clearer for new CSE hires later
- Added page for support onboarding

@emchap since this is my first time adding a new page and making changes this way, can you do a quick review and just make sure I haven't broken anything? Thank you!

### CODE REVIEW

**Comment by virginiaulrich** [comment]
Based on the failures, I suspect I did something quite wrong....

**Comment by virginiaulrich** [comment]
@emchap truly ready for your review now. 🙏🏻

File: handbook/ce/index.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ We work with many other teams at Sourcegraph, including:
 
 ---
 
-* [Onboarding](onboarding.md)
+* [Customer Engineering Onboarding](onboarding.md)
+* [Customer Support Onboarding] (customer-support-onboarding.md)
```
Comment (emchap) [inline handbook/ce/index.md:16]: @virginiaulrich Need to remove the space between `] (` on line 16! :)

File: handbook/ce/customer-support-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Customer Support Onboarding (WIP)
+
+Currently a work-in-progress; expected to have a working version 1 by 2021-02-19
+
+Welcome to Sourcegraph, to the Customer Engineering department, and the Customer Support team! This document will guide you through customer support-specific onboarding. It builds on our company onboarding and focuses on everything you need to be successfull in this particular role.
```
Comment (emchap) [inline handbook/ce/customer-support-onboarding.md:5]: @virginiaulrich Should that be "successful" one l?

**Review by emchap — CHANGES_REQUESTED** [review]
@virginiaulrich One markdown tweak and one spelling question—feel free to merge in after fixing the markdown, or I'm happy to re-review if you ping me. :)

File: handbook/ce/index.md (hunk 1 above)
Comment (virginiaulrich) [reply handbook/ce/index.md]: @emchap updated :)

File: handbook/ce/customer-support-onboarding.md (hunk 2 above)
Comment (virginiaulrich) [reply handbook/ce/customer-support-onboarding.md]: @emchap yup! Updated :)

**Review by emchap — APPROVED** [review]

**Comment by emchap** [comment]
@virginiaulrich lgtm! squash and merge, assuming tests pass.

**Pull request merged**